frontend/src/components/Pages/InspectionPage/InspectionSection.tsx

```diff
@@ -202,20 +374,79 @@ export function InspectionSection({ refreshInterval }: RefreshProps) {
                 )}
-            </StyledContent>
+            </StyledDeckOverview>
             {isDialogOpen && (
                 <ScheduleMissionDialog
                     missions={selectedMissions!}
                     refreshInterval={refreshInterval}
-                    closeDialog={() => setisDialogOpen(false)}
+                    closeDialog={closeDialog}
+                    setMissions={setSelectedMissions}
+                    unscheduledMissions={unscheduledMissions!}
+                    isAlreadyScheduled={isAlreadyScheduled}
                 />
             )}
         </>
     )
 }
+
+interface ICardMissionInformationProps {
+    deckId: string
+    deckMissions: DeckMissionType
+}
+
+function CardMissionInformation({ deckId, deckMissions }: ICardMissionInformationProps) {
+    const { TranslateText } = useLanguageContext()
+
+    var colorsCount: DeckMissionCount = {
+        red: { count: 0, message: 'Must be inspected this week' },
+        orange: { count: 0, message: 'Must be inspected within two weeks' },
+        green: { count: 0, message: 'Up to date' },
+        grey: { count: 0, message: '' },
+    }
+
+    deckMissions[deckId].inspections.forEach((inspection) => {
+        if (!inspection.deadline) {
+            if (!inspection.missionDefinition.lastRun && inspection.missionDefinition.inspectionFrequency) {
+                colorsCount['red'].count++
+            } else {
+                colorsCount['green'].count++
+            }
+        } else {
+            const dealineColor = getDeadlineInspection(inspection.deadline)
+            colorsCount[dealineColor!].count++
+        }
+    })
+
+    return (
+        <StyledMissionInspections>
+            {Object.keys(colorsCount)
+                .filter((color) => colorsCount[color].count > 0)
+                .map((color) => (
+                    <StyledMissionComponents key={color}>
+                        <StyledCircle style={{ background: color }} />
+                        <Typography color={tokens.colors.text.static_icons__secondary.rgba}>
+                            {colorsCount[color].count > 1 &&
+                                colorsCount[color].count +
+                                    ' ' +
+                                    TranslateText('Missions').toLowerCase() +
+                                    ' ' +
+                                    TranslateText(colorsCount[color].message).toLowerCase()}
+                            {colorsCount[color].count === 1 &&
+                                colorsCount[color].count +
+                                    ' ' +
+                                    TranslateText('Mission').toLowerCase() +
+                                    ' ' +
+                                    TranslateText(colorsCount[color].message).toLowerCase()}
+                        </Typography>
+                    </StyledMissionComponents>
+                ))}
+        </StyledMissionInspections>
+    )
+}
```

Comment:
This file is very long. Consider if it makes sense to move some of the content into a separate file